### PR TITLE
Implementing instance groups api coverage #61

### PR DESCRIPTION
**Pull request by mlazarov**

### CODE REVIEW

**Comment by Temikus** [comment]
I see some issues with it currently, like the body of the requests pointing at your own project, thus making the operations fail on any other account/zone:
```
pry(main)> connection.add_instance_group_instance("vagrant-group", "europe-west1-d", "fog-instance-groups-test")
Fog::Errors::NotFound: The resource 'projects/chaostest-1' was not found
from /Users/temikus/Code/vagrant-dev/fog-google/lib/fog/google.rb:230:in `build_excon_response'
```
How much have you tested this change? I would suggest writing tests usually, but with the current test model transition, this will not be easy, see: https://github.com/fog/fog-google/issues/50

Additionally, is it possible to cover the whole [API scope](https://cloud.google.com/compute/docs/instance-groups/)? Add resizing, etc.?

File: lib/fog/google/requests/compute/remove_instance_group_instance.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          parameters = {
+            'project' => @project,
+            'instanceGroup' => group_name,
+            'zone' => zone
+          }
+
+          body_object = {
+            "instances" => [
+              {
+                "instance" => "https://www.googleapis.com/compute/v1/projects/chaostest-1/zones/us-central1-a/instances/inst-1\n"
```
Comment (icco) [inline lib/fog/google/requests/compute/remove_instance_group_instance.rb]: Change these to use paramaters.

**Comment by mlazarov** [comment]
@Temikus, @icco thank you both for the comments. The issues related to the fixed instance params pointing to my test project are now fixed and should work without problem.

I'm testing these requests using simsple test script:

```
require 'rubygems'
require 'pp'

$LOAD_PATH << "lib"

require 'fog/google'
Fog.credential = :test


connection =  Fog::Compute.new({ :provider => "Google" })

zone = "us-central1-a"
group = "demo-"+Fog::Mock.random_numbers(19).to_s
inst = "inst-1"

print "Create new Group: #{group}\n"
resg = connection.instance_groups.create({ :name => group, :zone => zone, :description => "opisanie :)"})

print "Get the new group:\n"
res = connection.instance_groups.get(group)

print "Get All groups:\n"
res = connection.instance_groups.all({:zone => zone})

print "Add instance to group:\n"
res = connection.instance_groups.add_instance({:group => group, :instance => inst, :zone => zone})

#print "Remove instance from group:\n"
#res = connection.instance_groups.remove_instance({:group => group, :instance => inst, :zone => zone})

print "Destroy group:\n"
####res = connection.instance_groups.destroy({:name => group, :zone => zone})
resg.destroy

print "\nDONE\n"
```

Martin

**Comment by icco** [comment]
LGTM.

**Pull request merged**